[Bug]: Spellcheck makefile target is failing on Fedora linux

### Is there an existing issue already for this bug?

- [X] I have searched for an existing issue, and could not find anything. I believe this is a new bug.

### I have read the troubleshooting guide

- [X] I have read the troubleshooting guide and I think this is a new bug.

### I am running a supported version of CloudNativePG

- [X] I have read the troubleshooting guide and I think this is a new bug.

### Contact Details

_No response_

### Version

1.23.0

### What version of Kubernetes are you using?

1.30 (unsuppprted)

### What is your Kubernetes environment?

Other

### How did you install the operator?

Other

### What happened?

```$ make spellcheck
docker run --rm -v /home/leonardo/prj/cloudnative-pg:/tmp jonasbn/github-action-spellcheck:0.36.0

Using pyspelling on configuration outlined in >spellcheck.yaml<
Checking files matching specified outlined in >spellcheck.yaml<
----------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/bin/pyspelling", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pyspelling/__main__.py", line 37, in main
    return run(
           ^^^^
  File "/usr/local/lib/python3.12/site-packages/pyspelling/__main__.py", line 64, in run
    for results in spellcheck(
  File "/usr/local/lib/python3.12/site-packages/pyspelling/__init__.py", line 703, in spellcheck
    config = util.read_config(config_file)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pyspelling/util/__init__.py", line 186, in read_config
    raise ValueError(
ValueError: Unable to find or load pyspelling configuration from spellcheck.yaml, for more details on configuration please read https://facelessuser.github.io/pyspelling/configuration/
::error title=Spelling errors::Files in repository contain spelling errors
make: *** [Makefile:237: spellcheck] Error 1
```

### Cluster resource

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/Makefile
+++ b/Makefile
@@ -234,7 +234,7 @@ shellcheck: ## Shellcheck for the hack directory.
 	}
 
 spellcheck: ## Runs the spellcheck on the project.
-	docker run --rm -v $(PWD):/tmp jonasbn/github-action-spellcheck:$(SPELLCHECK_VERSION)
+	docker run --rm -v $(PWD):/tmp:Z jonasbn/github-action-spellcheck:$(SPELLCHECK_VERSION)
 
 woke: ## Runs the woke checks on project.
 	docker run --rm -v $(PWD):/src -w /src getwoke/woke:$(WOKE_VERSION) woke -c .woke.yaml

--- a/Makefile
+++ b/Makefile
@@ -237,7 +237,7 @@ spellcheck: ## Runs the spellcheck on the project.
 	docker run --rm -v $(PWD):/tmp:Z jonasbn/github-action-spellcheck:$(SPELLCHECK_VERSION)
 
 woke: ## Runs the woke checks on project.
-	docker run --rm -v $(PWD):/src -w /src getwoke/woke:$(WOKE_VERSION) woke -c .woke.yaml
+	docker run --rm -v $(PWD):/src:Z -w /src getwoke/woke:$(WOKE_VERSION) woke -c .woke.yaml
 
 wordlist-ordered: ## Order the wordlist using sort
 	LANG=C LC_ALL=C sort .wordlist-en-custom.txt > .wordlist-en-custom.txt.new && \

